src/pages/eol.js

```diff
@@ -0,0 +1,199 @@
+import React, { useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import PageTitle from '../components/PageTitle';
```

Comment:
let's keep all the 3rd party imports at the top then follow those by local imports

Reply:
aka - move this to line 12